tests/baselines/reference/strictFunctionTypes1.js

```diff
@@ -82,7 +82,7 @@ declare function fs(x: string): void;
 declare function fx(f: (x: "def") => void): void;
 declare const x1: (x: string) => void;
 declare const x2 = "abc";
-declare const x3: string;
+declare const x3: "def" | "abc";

```

Comment:
Seems like another case of differing widening in the declaration emit?

Reply:
This seems like it is wrong in the current emit. Hovering over the type will show `"def" | "abc"`. This seems because the type was widened in `createTypeOfDeclaration` regardless of if it was actually needed or not.

```ts

    function createTypeOfDeclaration(declarationIn: AccessorDeclaration | VariableLikeDeclaration | PropertyAccessExpression, enclosingDeclaration: Node, flags: NodeBuilderFlags, internalFlags: InternalNodeBuilderFlags, tracker: SymbolTracker) {
        const declaration = getParseTreeNode(declarationIn, isVariableLikeOrAccessor);
        if (!declaration) {
            return factory.createToken(SyntaxKind.AnyKeyword) as KeywordTypeNode;
        }
        // Get type of the symbol if this is the valid symbol otherwise get type at location
        const symbol = getSymbolOfDeclaration(declaration);
        const type = symbol && !(symbol.flags & (SymbolFlags.TypeLiteral | SymbolFlags.Signature))
            ? getWidenedLiteralType(getTypeOfSymbol(symbol))
            : errorType;

        return nodeBuilder.serializeTypeForDeclaration(declaration, type, symbol, enclosingDeclaration, flags | NodeBuilderFlags.MultilineObjectLiterals, internalFlags, tracker);
    }
    
    ```

Reply:
Removed the change.